web/src/settings_users.js

```diff
@@ -263,12 +263,16 @@ function bot_info(bot_user_id) {
 }
 
 function get_last_active(user) {
-    const last_active_date = presence.last_active_date(user.user_id);
-
-    if (!last_active_date) {
-        return $t({defaultMessage: "Unknown"});
+    if (user_setting_presence_info[String(user.user_id)]) {
+        const userPresence = user_setting_presence_info[String(user.user_id)];
```

Comment:
As I noted earlier, you should stick to snake_cave. So `user_presence`.

Reply:
Got it, I'll update the variable name to `presence_row` as suggested. Thanks for pointing it out!